packages/azure-http-specs/package.json

```diff
@@ -1,10 +1,9 @@
 {
   "name": "@azure-tools/azure-http-specs",
-  "version": "0.1.0",
+  "version": "0.1.0-alpha.0",
   "description": "Spec scenarios and mock apis",
   "main": "dist/index.js",
   "type": "module",
-  "private": true,
   "scripts": {
     "watch": "tsc -p ./tsconfig.build.json --watch",
     "build": "tsc -p ./tsconfig.build.json",
```

Comment:
can't comment there and not sure how its not failing, but the `test` script having `exit 1` was making things fail locally when running pnpm test(same applies to the one in typespec repo)

Reply:
@timotheeguerin  Removed the exit code 1 from the test sccript. I will do the same for typespec repo also in a separate PR.